# request
float32 x
float32 y
float32 z
float32 theta
---
# result
bool success
---
# feedback
int32 status